source/getting-started/onboarding.markdown

```diff
@@ -5,19 +5,43 @@ description: "Instructions to get Home Assistant configured."
 
 Alright, you made it here. The tough part is done.
 
-With Home Assistant installed, it's time for doing the initial configuration. Here you will create the owner account of Home Assistant. This account will be an administrator and will always be able to change everything. Enter a name, username, password and click on "create account".
+With Home Assistant installed, it's time to configure it. Here you will create the owner account of Home Assistant. This account will be an administrator and will always be able to change everything. Enter a name, username, password and click on "create account".
 
-Now you're brought to the main screen of Home Assistant: the states screen. The states screen will show all your devices. So let's get that screen filled up!
+<p class='img'>
+<img src='/source/getting-started/screenshots/01 - username.png' />
+Set your username and password.
+</p>
+
+Next, you can enter a name for your home and set your location and unit system. Click DETECT to find your location and set your time zone and unit system based on that location. If you'd rather not send your location, you can set these values manually.
+
+<p class='img'>
+<img src='/source/getting-started/screenshots/03 - location.png' />
+Set your location, time zone, and unit system.
+</p>
+
+Once you are done, click Next. In this screen, Home Assistant will show any devices that it has discovered on your network. Don't be alarmed if you see fewer items than what is shown below; you can always manually add devices later.
+
+<p class='img'>
+<img src='/source/getting-started/screenshots/04 - devices.png' />
+Discovery of devices on your network.
+</p>
+
+Finally, click Finish. Now you're brought to the main screen of Home Assistant called Lovelace. This screen will show all of your devices. So let's get that screen filled up!
```

Comment:
The screen is not called Lovelace. Lovelace is only a part of the full frontend.